src/utilities/strategy-helpers/blue-green-helper.ts

```diff
@@ -167,14 +146,25 @@ export function getManifestObjects (filePaths: string[]): any {
         });
     })
 
+    const serviceEntityList = [];
+    const unroutedServiceEntityList = [];
+
+    allServiceEntityList.forEach((serviceObject) => {
+        if (isServiceRouted(serviceObject, deploymentEntityList)) {
+            serviceEntityList.push(serviceObject);
+        } else {
+            unroutedServiceEntityList.push(serviceObject);
+        }
+    })
+
     let serviceNameMap = new Map<string, string>();
     // find all services and add their names with blue green suffix
     serviceEntityList.forEach(inputObject => {
         const name = inputObject.metadata.name;
         serviceNameMap.set(name, getBlueGreenResourceName(name, GREEN_SUFFIX));
     });
      
-    return { serviceEntityList: serviceEntityList, serviceNameMap: serviceNameMap, deploymentEntityList: deploymentEntityList, ingressEntityList: ingressEntityList, otherObjects: otherEntitiesList };
+    return { serviceEntityList: serviceEntityList, serviceNameMap: serviceNameMap, unroutedServiceEntityList: unroutedServiceEntityList, deploymentEntityList: deploymentEntityList, ingressEntityList: ingressEntityList, otherObjects: otherEntitiesList };
 }
 
 export function isServiceRouted(serviceObject: any[], deploymentEntityList: any[]): boolean {
```

Comment:
Is `UnsetsClusterSpecficDetails` being used anywhere?

Reply:
yes in `fetchResource`